homeassistant/components/sensor/ebusd.py

```diff
@@ -0,0 +1,103 @@
+"""
+Support for Ebusd daemon for communication with eBUS heating systems.
+
+For more details about ebusd deamon, please refer to the documentation at
+https://github.com/john30/ebusd
+"""
+
+import logging
+
+from homeassistant.const import (
+    STATE_ON, STATE_OFF)
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['ebusd']
+
+DOMAIN = 'ebusd'
+DATA_EBUSD = 'EBUSD'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Ebus sensor."""
+    ebusd_api = hass.data[DATA_EBUSD]
+    monitored_conditions = discovery_info['monitored_conditions']
+    name = discovery_info['client_name']
+
+    dev = []
+    for condition in monitored_conditions:
+        dev.append(Ebusd(
+            ebusd_api, discovery_info['sensor_types'][condition], name))
+
+    add_entities(dev, True)
+
+
+def timer_format(string):
```

Comment:
If there is a special time string returned from the device, the time formatter should be part of the ebusdpy library.